bin/oneoff/backfill_data/principalapproval1920_application_pay_fee_changes.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+ActiveRecord::Base.transaction do
+  total_updated = 0
+  errors = []
+  puts "Backfilling :pay_fee question changes..."
+
+  Pd::Application::PrincipalApproval1920Application.find_each do |application|
+    print '.'
+    pay_fee = application.form_data_hash["payFee"]
+
+    if pay_fee == 'Yes, my school or teacher will be able to pay the full program fee.'
+      new_pay_fee = 'Yes, my school will be able to pay the full program fee.'
+    elsif pay_fee == 'No, my school or teacher will not be able to pay the program fee. We would like to be considered for a scholarship.'
+      new_pay_fee = 'No, my school will not be able to pay the program fee. We would like to be considered for a scholarship.'
+    else
+      next
+    end
+
+    application.update_form_data_hash({"payFee": new_pay_fee})
+    if application.save
```

Comment:
A quick note that when running this on production to backfill on 5/10/19, this was changed to `save(validate: false)`.  Of 2926 principal approval applications, 29 failed validations, apparently each because they didn't have a linked `teacher_application`.  The backfill succeeded for all when we skipped validations.